datameta/api/apikeys.py

```diff
@@ -42,13 +45,35 @@ class UserSession:
 
     def __json__(self, request: Request) -> dict:
         return {
+                "apiKeyId": self.apikey_id,
                 "userId": self.user_id,
                 "email": self.email,
                 "token": self.token,
                 "label": self.label,
-                "expiresAt": self.expires_at
+                "expiresAt": str(self.expires_at) if self.expires_at else None
             }
 
+
+class ApiKeyLabels:
+    """ApiKeyLabels container for OpenApi communication"""
+
+    apikeys: List[dict]
+
+    def __init__(self, user:models.User):
+        self.apikeys = [
+            {
+                "apiKeyId": str(key.uuid),
+                "label": key.label,
+                "expiresAt": str(key.expires) if key.expires else None,
```

Comment:
Let's be specific about how we want to string-represent datetimes and the ISO 8601 representation is what's currently used in all other datetime string conversions in the code
```suggestion
                "expiresAt": key.expires.isoformat() if key.expires else None,
```

The same applies to line 105 which I can't comment on because it was already committed previosly

Reply:
Good Idea, will adapt